modules/distribution/resources/api_templates/velocity-template_synapse-config_non-versioned-webapp.xml

```diff
@@ -0,0 +1,110 @@
+<!--
+ ~ Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
fixed in commit cc7f512ebecdfaa5b7be5b4ea8b38b0bddef1475 of PR https://github.com/wso2/product-app-manager/pull/303